dspace-server-webapp/src/test/java/org/dspace/app/rest/BitstreamRestControllerIT.java

```diff
@@ -425,5 +514,127 @@ private int getNumberOfPdfPages(byte[] content) throws IOException {
         }
     }
 
+    @Test
+    public void getBitstreamFormat() throws Exception {
+
+        getClient().perform(get("/api/core/bitstreams/" + bitstream.getID() + "/format"))
+                .andExpect(status().isUnauthorized());
 
+        getClient(getAuthToken(eperson.getEmail(), password))
+                .perform(get("/api/core/bitstreams/" + bitstream.getID() + "/format"))
+                .andExpect(status().isForbidden());
+
+        context.turnOffAuthorisationSystem();
+
+        resourcePolicy = resourcePolicyService.create(context);
+        resourcePolicy.setEPerson(eperson);
+        resourcePolicy.setAction(READ);
+        resourcePolicy.setdSpaceObject(bitstream);
+
+        context.commit();
```

Comment:
This should not be a `commit()`.  It should instead be a `context.restoreAuthSystemState();`